demo/api/src/products/entities/manufacturer.entity.ts

```diff
@@ -32,9 +32,9 @@ export class AlternativeAddress {
 @ObjectType()
 @InputType("AddressInput")
 export class Address extends AlternativeAddress {
+    @IsNullable()
     @Property({ type: "json", nullable: true })
     @Field(() => AlternativeAddress, { nullable: true })
-    @IsObject()
```

Comment:
Why do you remove this?

Reply:
because this does not allow null-value.

Reply:
What about using both validators together? Then it would be either null or object. Now you don't have any validation.